src/main/kotlin/io/github/inductiveautomation/kindling/core/Kindling.kt

```diff
@@ -48,12 +51,15 @@ data object Kindling {
         listOf(16, 32, 44, 64, 128, 150, 256, 512, 1024).map { dim ->
             BufferedImage(dim, dim, BufferedImage.TYPE_INT_ARGB).apply {
                 val g = createGraphics()
-                svgDocument.render(null, g)
+                g.setRenderingHint(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON)
+                svgDocument.render(null, g, ViewBox(dim.toFloat(), dim.toFloat()))
```

Comment:
Crucial! Gotta AA + render with the right viewbox for these icons to work right.